.rubocop.yml

```diff
@@ -11,9 +11,13 @@ Style/Documentation:
 
 Metrics/MethodLength:
   Max: 15
+  Exclude:
+    - 'lib/keyrod/oidc_client.rb'
 
 Metrics/AbcSize:
   Max: 20
+  Exclude:
```

Comment:
Ditto :smile:

Reply:
For some reason this metric counts string interpolation aswell, which I need for logging. I will keep this file excluded for this metric for now.

Reply:
No, you won't. It's a new class with simple code and simple functionality. If its ABC size is too big it's because it's poorly written. If the ABC size is too big dividing method into multiple smaller methods usually helps.